From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Milan Crha <mcrha@redhat.com>
Date: Tue, 7 Jul 2020 11:29:48 +0200
Subject: [PATCH] eds:add-persona: Test fails when eds is compiled with
 libphonenumber

When eds is built with libphonenumber support it also returns "X-EVOLUTION-E164"
parameter on TEL attributes, which breaks the eds:add-persona test, because it
expects exact parameters being returned from eds.

Closes https://gitlab.gnome.org/GNOME/folks/-/merge_requests/40
---
 tests/eds/add-persona.vala | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/eds/add-persona.vala b/tests/eds/add-persona.vala
index ce64ce19..4483133d 100644
--- a/tests/eds/add-persona.vala
+++ b/tests/eds/add-persona.vala
@@ -400,6 +400,9 @@ public class AddPersonaTests : EdsTest.TestCase
 
       foreach (var phone_fd in i.phone_numbers)
         {
+          /* Can be returned, when the evolution-data-server is compiled with the libphonenumber support */
+	  phone_fd.remove_parameter_all ("x-evolution-e164");
+
           var phone_1 = new PhoneFieldDetails (this._phone_1);
           phone_1.set_parameter (AbstractFieldDetails.PARAM_TYPE,
               this._phone_1_type);
